src/Services/Contracts/ReverseGeocodes/IBigDataCloudReverseGeocodeService.cs

```diff
@@ -1,9 +1,11 @@
+using PhotoCli.Models.ReverseGeocode;
+
 namespace PhotoCli.Services.Contracts.ReverseGeocodes;
 
 public interface IBigDataCloudReverseGeocodeService
 {
 	Task<IEnumerable<string>> Get(Coordinate coordinate, string? language, IEnumerable<int> adminLevels);
 
```

Comment:
There was empty line that I forgot previously. As you touch this file, can you remove these empty line?